config/500-webhook-configration.yaml

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 The Knative Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
Can we use `v1` for all webhook configurations?

Reply:
ok.
I get the same warning on `apiextensions.k8s.io/v1beta1` in the CDR yaml. should I change it there as well?

Reply:
yes

Reply:
BTW, knative is using v1beta1 CDRs...
see: https://knative.dev/docs/install/any-kubernetes-cluster/